docs/instruments/ZJSONInstrument_v0.0.1.md

```diff
@@ -551,6 +534,14 @@ represented on the front-end by an input element. Its value is instead calculate
     } 
 }
 ```
+`options.scoringFormula`: Required string. An expression of a logical formula to be calculated by LORIS LParse. The formula is evaluated to true or false if a boolean score, or a numeric if a number score. The result is dependent and defined by the value of this key.
+
+`options.readonly`: Required boolean.
+                    True or false that the field is read only i.e. not editable on the front-end but
+                    submittable on save. This key is different to `rules.disableIf` in that it is not
+                    dependent on a condition, and a 'disabled' field is not submittable.
+
+`options.hideInSurvey`: Required boolean. True or false that this score field will be hidden and not displayed when the instrument is in survey mode.
```

Comment:
@zaliqarosli I think in some cases, we need to hide other fields also in Survey mode apart from score fields.